.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,55 @@
+### Instructions
+
+- Determine whether your issue is a **bug report**, a **feature request**, or a **documentation request** (for tool gatkdocs only; for forum docs see [the support issues repo](https://github.com/broadinstitute/dsde-docs/issues));
```

Comment:
Maybe a few sentences of prefix before diving in?  Something like `The github issue tracker is for bug reports, feature requests, and API documentation requests.  General questions about how to use the GATK, how to interpret the output, etc should be asked on the official support forum. "